exawind/Pazy_Wing/README.md

```diff
@@ -0,0 +1,177 @@
+
+The `plotting` directory also contains the files needed to reproduce the flutter boundary figure.  For this, you will need to do a parameter sweep (the `setup.sh` script will help with this), and run cases for several windspeeds near the flutter onset speed for each AoA (3, 5, and 7 degrees).  From the deflection history (obtained from the `post.py` script), one can determine whether each case meets the criteria for "stable", "unstable", or "flutter", as discussed in the results section below.  These speeds can be added to the files `nalustable.dat`, `naluunstable.dat`, and `naluflut.dat`.  Then, the figure may be reproduced by running `python3 plotflut.py`.
+
+## Code Performance
+
+For these simulations, 90% of the cores were allocated to Nalu-Wind, and the remainder to AMR-Wind.  However, the timings indicated that adding yet more cores to Nalu-Wind would yield a speed improvement in this case.  Future studies are planned to determine the optimal resource balance.
+
+| Property              | Value                  |
+| :---                  | ---                    |
+| Wall-Clock Time       | 48 hours               |
+| Simulated Time        | 2.09 seconds           |
+| Number of Timesteps   | 34,380                 |
+| Number of CPUs        | 560                    |
+| Number of HPC Nodes   | 5                      |
+| Nalu Cells per CPU    | ~15,600                |
+
+The job was run on the Sandia Flight HPC cluster, which has the following specifications:
+
+| Parameter           | Value |
+|---                  |---  |
+| CPU processor type  | Intel(R) Xeon(R) Platinum 8480+ |
+| CPU processor speed | 3800 Mhz |
+| Node interconnects  | Cornelis Omni-Path high-speed interconnect |
+
+## Results
+
+**Static Deflection Results**
+
+<img src="figures/combined.png" border="0" alt=""/>
+
+For wind velocities below the flutter onset boundary, the wing deformation approaches a steady state after the initial oscillatory transients decay.  The final tip deflection is a function of the freestream velocity and the root angle of attack.  Comparison data is available for three angles of attack (3, 5 and 7 degrees), and wind speeds between 15 m/s and the flutter onset speed, which depends on the angle of attack.  For each AoA, comparisons are shown above.  The two black curves are experimental data from [1].  The solid black curve is from a sweep of dynamic pressure (while holding AoA constant), and the dashed black curve is from a sweep of AoA (while holding dynamic pressure constant).  The solid black line is used as the reference for comparison in [2], and we likewise treat it as the more reliable of the two experimental data sets.  The grey band indicates the spread in computational comparison data given in [2].
+
+**Flutter Results**
+
+<img src="figures/flut_figs_350.png" border="0" alt="" width="900"/>
```

Comment:
My apologies, this is a limitation of the deploy script I wrote. We'll need to switch it to use the markdown method of including images, i.e.,
```markdown
![Flutter](figures/flut_figs_350.png)
```
Otherwise it won't get copied over to the website directory.

Reply:
Done!